src/app/helptext/task-calendar/replication/replication-wizard.ts

```diff
@@ -54,25 +54,25 @@ export default {
  the selected dataset snapshots.'),
 
     encryption_placeholder: T('Encryption'),
-    encryption_tooltip: T(''),
+    encryption_tooltip: T('Checking this box, enables additional options for encryption replication'),
 
     encryption_key_generate_placeholder: T('Generate Encryption Key'),
-    encryption_key_generate_tooltip: T(''),
+    encryption_key_generate_tooltip: T('If Hex key type is selected, this will autogenerate an encryption key'),
 
     encryption_key_hex_placeholder: T('Encryption Key'),
-    encryption_key_hex_tooltip: T(''),
+    encryption_key_hex_tooltip: T('Uncheck Generate Encryption Key to import your custom Hex key'),
```

Comment:
Cleanup suggestion:
"Unset <i>Generate Encryption Key</i> to instead import a custom Hex key."